### ISSUE TITLE
Add Item - 'Add Another File' button fails to update screen reader users

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1136]
"Description: On the ‘Add an Item’ page, in the ‘Files’ tab, there is a button labeled, ‘Add Another File’. If users are utilizing a screen reader and users select the button, the screen reader will not give the user a status update if the button is activated or not. Visual users are able to tell that a file row has been added, but non-visual screen reader users would need to navigate to and count the number of rows to get the same information.

Expected Results: Screen reader users should be able to identify if buttons/components on a page have been activated. As the user adds files, "# files added" would be announced to inform the user about the number of files that have been added.

Actual Results: Upon activation of the ‘Add Another File’ button, screen reader users are left guessing if it was activated or not."

4.1.3 Status Messages; Level AA; WCAG 2.1
